### ISSUE TITLE
Auto refresh room list after leaving room

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#320]
```
What steps will reproduce the problem?
1. Join room
2. Leave room
3. I have to click "Refresh" button, but it should be auto refresh

What is the expected output? What do you see instead?

I have to click "Refresh" button, but it should be auto refresh


What version of the product are you using? On what operating system?

Hedgewars 0.9.17, 
System: Kubuntu 11.10

Please provide any additional information below.


```

-----
Original issue reported on code.google.com by `blackmet...@o2.pl` on 1 Dec 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `kyberneticist@gmail.com` on 2 Dec 2011 at 1:48
* Added labels: **Type-Enhancement**
* Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c6355e1602e6.
```

-----
Original comment by `sheepyluva` on 7 Dec 2011 at 11:21
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 677b96d13e1f.
```

-----
Original comment by `unC0Rr` on 30 Dec 2011 at 8:29

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `bmarti...@gmail.com` on 27 Nov 2012 at 4:29
* Changed state: **Verified**
